app/api/V1/views.py

```diff
@@ -0,0 +1,76 @@
+from flask_restful import Api, Resource 
+from flask import jsonify, make_response, request
+import datetime
+
+from .models import IncidenceModel
+
+
+class RedFlags(Resource):
+    def __init__(self):
+        self.db = IncidenceModel()
+         
+
+    def post(self): 
+        data = request.get_json(force=True)
+        CreatedBy = data['CreatedBy']
+        IncidenceType = data['IncidenceType']
+        Location = data['Location']
+        Status = data['Status']
+        Comment = data['Comment'] 
+         
+        resp = self.db.save(CreatedBy, IncidenceType, Location, Status, Comment)
```

Comment:
Here, `resp` seems to return the whole self.db instead of the specific created incident.
Also, we don't need a success message since we already have the status code